config/helm/chart/default/templates/Common/csi/daemonset.yaml

```diff
@@ -285,9 +285,16 @@ spec:
       {{- if .Values.csidriver.nodeSelector }}
       nodeSelector: {{- toYaml .Values.csidriver.nodeSelector | nindent 8 }}
       {{- end }}
-      {{- if .Values.csidriver.tolerations }}
-      tolerations: {{- toYaml .Values.csidriver.tolerations | nindent 8 }}
-      {{- end }}
+      tolerations:
+        {{- if .Values.csidriver.tolerations }}
+        {{- toYaml .Values.csidriver.tolerations | nindent 8 }}
+        {{- end }}
+        - key: kubernetes.io/arch
```

Comment:
```
    - effect: NoSchedule
      key: ToBeDeletedByClusterAutoscaler
      operator: Exists
```
maybe having this here as well would make sense ? (its in the values.yaml, the other 2 in the values.yaml should be configurable)

Reply:
done